code/obj/machinery/launcherloader.dm

```diff
@@ -19,6 +19,8 @@
 	var/operating = 0
 	var/driver_operating = 0
 	var/trash = 0
+	/// Amount of time *in seconds* before connected blast doors should close
```

Comment:
```suggestion
	/// Amount of time before connected blast doors should close
```

Reply:
It's multiplied by seconds, why is it bad to specify seconds